src/main/java/teammates/ui/webapi/GetCoursesAction.java

```diff
@@ -49,7 +48,6 @@ public JsonResult execute() {
     private JsonResult getStudentCourses() {
         List<CourseAttributes> courses = logic.getCoursesForStudentAccount(userInfo.id);
         CoursesData coursesData = new CoursesData(courses);
-        coursesData.getCourses().forEach(CourseData::hideInformationForStudent);
```

Comment:
@wkurniawan07 Saw that `CourseData::hideInformationForStudent` was introduced as part of #10328, would it be ok to reveal this information now that we want to enable sorting of sessions by create time?

Should we instead only hide course deletion time instead of both deletion and creation time

Reply:
@Zxun2 As mentioned by @zhaojj2209 lets keep the hiding the course deleted time but expose the creation time for our purposes of sorting

Reply:
Noted! 🫡